src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -75,7 +84,8 @@ public static void check(
     final AnnotationHolder holder
   ) {
     final ResultHolder type1 = HaxeTypeResolver.getTypeFromTypeTag(tag, erroredElement);
-    final ResultHolder type2 = HaxeTypeResolver.getPsiElementType(initExpression);
+    final ResultHolder type2 = HaxeTypeResolver.getPsiElementType(initExpression, new HaxeResolver2Dummy());
```

Comment:
I'm not sure why you would add a dummy argument to the resolver; I haven't gotten that far yet.  I'd prefer a new entry point to the resolver than having to know to add a dummy.  (Casual hackers aren't going to know to look for a HaxeResolver2Dummy() class in the first place.  They'll figure it out by looking for other examples, but why make the obstacle in the first place?)